src/cuda/host.rs

```diff
@@ -63,6 +63,45 @@ pub struct HostNet {
   rwts: u64,
 }
 
+impl HostNet {
+  pub fn to_runtime_net(self) -> run::Net {
+    fn wire_to_rdex(wire: Wire) -> (run::Ptr, run::Ptr) {
+      (run::Ptr((wire & 0xFFFFFFFF) as Val), run::Ptr((wire >> 32) as Val))
+    }
+
+    fn node_to_pair(node: Node) -> (run::Ptr, run::Ptr) {
+      let Node { ports: [a, b] } = node;
+      (run::Ptr(a), run::Ptr(b))
+    }
+
+    let mut rdex = Vec::new();
+
+    for i in 0..BAGS_SIZE {
+      let wire: u64 = self.bags[i as usize];
+      if i % RBAG_SIZE == 0 && wire > 0 {
+        rdex.push(wire_to_rdex(wire));
+      } else if i % RBAG_SIZE >= 1 {
+        let (a, b) = wire_to_rdex(wire);
+        if a.0 != 0 || b.0 != 0 {
+          rdex.push((a, b));
+        }
+      }
+    }
+
+    let data = self.heap.into_iter().map(|node| node_to_pair(*node)).collect();
+
+    run::Net {
+        rdex,
+        heap: run::Heap::from_data(data),
+        anni: 0,
+        comm: 0,
+        eras: 0,
+        dref: 0,
+        oper: self.rwts.try_into().unwrap(),

```

Comment:
we don't have detailed statistics on the gpu, right? I think then you should add a comment for this workaround